lib/hsa/hsaBeamformHFBZeroData.cpp

```diff
@@ -0,0 +1,41 @@
+#include "hsaBeamformHFBZeroData.hpp"
+
+using kotekan::bufferContainer;
+using kotekan::Config;
+
+REGISTER_HSA_COMMAND(hsaBeamformHFBZeroData);
+
+hsaBeamformHFBZeroData::hsaBeamformHFBZeroData(Config& config, const string& unique_name,
+                                               bufferContainer& host_buffers,
+                                               hsaDeviceInterface& device) :
+    hsaSubframeCommand(config, unique_name, host_buffers, device, "hsaBeamformHFBZeroData", "") {
+    command_type = gpuCommandType::COPY_IN;
+
+    int block_size = config.get<int>(unique_name, "block_size");
+    int num_elements = config.get<int>(unique_name, "num_elements");
+    _num_blocks = (int32_t)(num_elements / block_size) * (num_elements / block_size + 1) / 2.;
+    output_len = 1024
+                 * 128 // No. of frequencies
+                 //* (_samples_per_data_set / _downsample_time)
```

Comment:
Remove this line?  Also shouldn't we be getting these values from the config? The 1024 and 128?